low-level/sensors/impl/generic/raid_integrity_data.py

```diff
@@ -105,6 +108,14 @@ def initialize(self, conf_reader, msgQlist, product):
         if self._scan_frequency < self.MIN_SCAN_FREQUENCY:
             self._scan_frequency = self.MIN_SCAN_FREQUENCY
 
+        sysfs_path = Conf.get(SSPL_CONF,
+                              f'{SYSTEM_INFORMATION}>{SYSFS_PATH}')
```

Comment:
SYSFS_PATH, I don't see it updated in config file. Will it be there already?

Reply:
yes its there in config already